gapis/server/server.go

```diff
@@ -29,6 +29,7 @@ import (
 	"github.com/google/gapid/core/app/benchmark"
 	"github.com/google/gapid/core/event/task"
 	"github.com/google/gapid/core/log"
+	dev "github.com/google/gapid/core/os/device"
```

Comment:
Without the named import, this conflicts with os/device/bind.
Unless I am missing something about how go works.
Basically it cannot find device.Android, but with the named import it can find dev.Android